mitosheet/mitosheet/api/get_code_snippets.py

```diff
@@ -63,4 +73,21 @@
 ]
 
 def get_code_snippets(params: Dict[str, Any], steps_manager: StepsManagerType) -> str:
-    return json.dumps(DEFAULT_CODE_SNIPPETS)
+        # If code snippet environment variables are set, then use them to get the code snippets. 
+        mito_config_code_snippets_version = os.environ.get(MITO_CONFIG_KEY_CODE_SNIPPETS_VERSION)
+        mito_config_code_snippets_url = os.environ.get(MITO_CONFIG_KEY_CODE_SNIPPETS_URL)
+        if mito_config_code_snippets_version == '1' and mito_config_code_snippets_url is not None:
+                
+                # Request the code snippets from the url
+                response = req.get(mito_config_code_snippets_url, verify=False)
+
+                if response.status_code == 200:
+                        # Parse the respone body into JSON 
+                        code_snippets = response.json()
```

Comment:
I think we should check that these have the correct format, no? Otherwise the frontend will likely crash. 

If they don't, we can raise a ValueError with that specific value that says they are undefined.